src/rpc/blockchain.cpp

```diff
@@ -194,7 +194,7 @@ UniValue waitfornewblock(const JSONRPCRequest& request)
 {
     if (request.fHelp || request.params.size() > 1)
         throw runtime_error(
-            "waitfornewblock\n"
+            "waitfornewblock <timeout>\n"
```

Comment:
Outside of the scope of this PR, but as we are here: timeout is int, in ms. And the default is false!?